services/api/src/main.py

```diff
@@ -44,10 +45,32 @@
 ENABLE_WZDX_FEATURES = os.environ.get("ENABLE_WZDX_FEATURES", "true") != "false"
 
 smtp_error_handler.configure_error_emails(app)
+errors.register_error_handlers(app)
 
 app.wsgi_app = Middleware(app.wsgi_app)
 api = Api(app)
 
+# /rsucounts
```

Comment:
Are these comments intended to show the different endpoint options?

Reply:
removed!